### DISCUSSION TITLE
live output from executed command on remote machine

### DISCUSSION THREAD

**Discussion by arabnejad** [ParallelSSH/ssh2-python#149] [Q&A] [answered]
Hi @pkittenis 

how can we have a live and stream output execution for the command that executed on the target host, 
this is my first time using `ssh2-python` package, so far, based on the examples I found, we fetch the output after it executed and the we can print it, 
is there any way that we can have live execution for the command ?

**Comment by pkittenis** [comment, marked as answer]
Thanks for the interest.

See [parallel-ssh](https://github.com/ParallelSSH/parallel-ssh) for a high level client.

It sounds like you are describing [non-blocking execute](https://github.com/ParallelSSH/ssh2-python/blob/master/examples/nonblocking_execute.py).
